master pull

nhendry05 opened the pull request.

DezzaPR0XY committed:
Update README.md

DezzaPR0XY committed:
Update README.md

DezzaPR0XY committed:
adding templates and css

DezzaPR0XY committed:
Assigned roles in README

DezzaPR0XY committed:
Merge pull request #1 from DezzaPR0XY/derek_branch

Assigned roles in README and added Server file

CausticFoxx reviewed (commented):
A few questions; Why are there 2 different `.py` files?  both are separate runnable flask apps.  In the `main.py` it looks like there is an old assignment commented out, was this for reference purposes? I would also suggest, in regards to the `server.py`, the sqlconnection code should not be embedded and should have its own py file and then be imported into the `server.py` file for readability, debugging, etc. You can scrap the HTML and CSS files as I have started writing them from scratch as we discussed. :)

nhendry05 commented:
I was actually just pulling what was on master into my own branch (but did it wrong). I just ended up cloning it. I already deleted everything off the main.py file and started my own stuff. I think it may have been Dereks old assignment? I'm not really sure so I just deleted it all. I also agree that sqlconnection should not be embedded. I didnt even look through the other files yet
